app/controllers/performance_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PerformanceController < ApplicationController
+  def index
+    @comments = Comment.all
```

Comment:
As discussed I've now extracted all the queries into the controller and removed this behemoth Comment.all call :elephant: